### ISSUE TITLE
Add content item for a collapsable HTML part

### ISSUE THREAD

**Issue by espaan** [zikula-modules/Content#110]
Sometimes it is very convenient to have a part of a page by default collapsed or at least with the option of collapsing/expading that part.

This could be done via Zikula.UI with the Panels,but might be overkill, easier would be via JQuery (or Prototype).

The default HTML content item could also be extended to be collapsible yes/no and if yes by default open/closed.

**Comment by Guite** [comment]
Maybe something like this would be appropriate:
https://getbootstrap.com/docs/3.3/javascript/#collapse-example-accordion

Implementation could be done similar to the `TabNavigation` content type.

**Comment by Guite** [comment]
I need something like this for a current project, so probably I can put this into 5.0.0 in the next days.

**Comment by Guite** [comment]
> a headline should be possible to add

No, I don't like that, because this would introduce another layer for translations.

**Comment by rallek** [comment]
That is true! I did not think about translation.
